app/cdap/components/WrangleHome/Components/OngoingDataExplorations/index.tsx

```diff
@@ -14,113 +14,173 @@
  * the License.
  */
 
-import React, { useEffect, useState } from 'react';
 import { Box } from '@material-ui/core/';
-import { generateDataForExplorationCard } from './utils';
 import MyDataPrepApi from 'api/dataprep';
+import { getCategorizedConnections } from 'components/Connections/Browser/SidePanel/apiHelpers';
+import {
+  IConnectionWithConnectorType,
+  IExistingExplorationCardsData,
+  IConnectionsList,
+  IMassagedObject,
```

Comment:
rename this it IMassagedData
do not use Object or Array name in interface names or variable and function names